README.md

```diff
@@ -42,7 +42,12 @@ represent the tiles.
 
 ## A1.1
 
-«replace this with your answer»
+S -> T
+S -> T S T
+T -> "r"
+T -> "g" 
+T -> "b"
+
```

Comment:
# 3

Very close, but there's no guarantee that both Ts in "T S T" will expand to the same color. Instead you need something like:

       S -> r | g | b
       S -> r  S  r
       S -> g  S  g
       S -> b  S  b